origami.json

```diff
@@ -5,11 +5,24 @@
     "support": "next.team@ft.com",
     "supportStatus": "experimental",
     "runServer": "true",
+    "demosDefaults": {
+        "sass": "demos/src/demo.scss",
+        "js": "demos/src/demo.js",
+        "documentClasses": "",
+        "dependencies": "o-fonts@^2.0.0,o-grid@^4.0.0"
+    },
     "demos": [
         {
-            "path": "/demos/light-signup-demo.html",
-            "expanded": true,
-            "description": ""
+            "name": "promo-image",
+            "description": "Light signup form with promo image",
+            "template": "demos/src/standard.html",
+            "path": "/demos/promo-image.html"
```

Comment:
Just for some clarity:
- Locally I removed the `path` lines as mentioned
- I ran `make demo` to check all was good...cool its all working
- I went to commit the changes...git said nothing had changed. huh?
  - I realised after a few minutes its because the lines are being added automatically.
  - Therefore in order to commit the changes I would need to commit before running `make demo`.